src/Apps/Order/Routes/__tests__/Shipping.test.tsx

```diff
@@ -157,27 +157,27 @@ describe("Shipping", () => {
 
     it("shows an error modal when there is an error from the server", () => {
       const component = getWrapper(testProps)
-      expect(component.find("ErrorModal").prop("show")).toBe(false)
+      expect(component.find("ErrorModal").props().show).toBe(false)
```

Comment:
Let's also use the `ErrorModal` vs `"ErrorModal"` so TS will be able to infer the props type and we'll get notified at compile time when the `ErrorModal` component is renamed or removed.